adafruit_httpserver.py

```diff
@@ -287,7 +288,10 @@ def route(self, path: str, method: str = "GET"):
 
             @server.route(path, method)
             def route_func(request):
+                raw_text = request.raw_request.decode("utf8")
+                print("Received a request of length", len(raw_test), "bytes")
```

Comment:
the variable created is `raw_text` but the one it tries to print is `raw_test` they'll need to match, text I think is more descriptive so:

```py
print("Received a request of length", len(raw_test), "bytes")
```

Reply:
thanks for spotting it!